app/core/file_processor/langchain_loader.py

```diff
@@ -26,7 +26,7 @@ def process_file(
         with custom handling for its format and contents.
         Implementations should try to fill as much additional information like links, media etc.
         label, when provided, should apply to all documents in the o/p list"""
-        name = kwargs.get("name", file_path)
+        name = kwargs.get("name", file_path.split(".")[-1])
```

Comment:
will this give us just the extension?
I guess you meant to split by `/`
 `file_path.split("/")[-1].split('.')[0]`

Reply:
Ooops, yep! Updated now - thanks.